### ISSUE TITLE
Introduce an option for table resize to return its values in pixels 

### ISSUE THREAD

**Issue by mabryl** [ckeditor/ckeditor5#16999]
## 📝 Provide a description of the improvement

Currently, when you resize the table columns via Table Column Resize, the values are returned as percentages.

Some integrators would prefer that these values be presented in pixels instead of percentages, as was the case in CKE4. 

## 📃 Other details

* Browser: cross-browser
* OS: cross-OS
* CKEditor version: N/A
* Installed CKEditor plugins: N/A

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.

**Comment by Mati365** [comment]
@mabryl Looks like we already have PR with exactly this feature: https://github.com/ckeditor/ckeditor5/pull/16104 but it was closed after discussions. 
fyi @arkflpc @niegowski @Witoso

**Comment by scofalik** [comment]
Yes, table sizes is a complex topic, where semantics clash with different CSS/HTML details. We decided to revisit it again at some point and spent appropriate time on it.

**Comment by andrzejkala** [comment]
This issues was resolved in [`v48.4.0`](https://github.com/ckeditor/ckeditor5/releases/tag/v48.4.0).

> The [Column resize](https://ckeditor.com/docs/ckeditor5/latest/features/tables/tables-resize.html) feature now supports widths in pixels, following the table width unit, and an exact column width can be set through the cell width field.
> 
> Widths set via the resize handle and the properties form now stay in sync.
